src/app.js

```diff
@@ -40,9 +42,18 @@ app.get("/organisation-type", (req, res) => {
 });
 
 app.get("/help-request", (req, res) => {
-  res.render("helpRequest.njk"); // checkbox
+  res.render("helpRequest.njk", { showError: false }); // checkbox
 });
 
 app.listen(port, () => {
   console.log(`Example app listening on port ${port}`);
 });
+
+app.post("/validate-help-request", (req, res) => {
+  const result = validateForm(req, res, "helpWithHint", "/service-description");
```

Comment:
You don't need to pass "res", it's not used in validateForm function.
Also because you're using only req for get the the value of the field ("req.body[formType]") in your function, you don't have to pass req and formType parameters. Use only req.body.helpWithHint
That will give you this call:
` validateForm(req.body.helpWithHint, "/service-description");` 
2 parameters: the value, and the redirection path.

Reply:
ok so I changed the arguments which I pass when calling the function , but do I need to change the parameters when declaring my function ?